source/objects/cableClass.m

```diff
@@ -0,0 +1,283 @@
+        hardStops               = struct(...
+            'upperLimitSpecify', 'off',...        % (`string`) Initialize upper stroke limit. ``  'on' or 'off' `` Default = ``off``.
+            'upperLimitBound', 1, ...             % (`float`) Define upper stroke limit in m or deg. Only active if `lowerLimitSpecify` is `on` `` Default = ``1``.
+            'upperLimitStiffness', 1e6, ...       % (`float`) Define upper limit spring stiffness, N/m or N-m/deg. `` Default = ``1e6``.
+            'upperLimitDamping', 1e3, ...         % (`float`) Define upper limit damping, N/m/s or N-m/deg/s.  `` Default = ``1e3``.
+            'upperLimitTransitionRegionWidth', 1e-4, ... % (`float`) Define upper limit transition region, over which spring and damping values ramp up to full values. Increase for stability. m or deg. ``Default = 1e-4``
+            'lowerLimitSpecify', 'off',...        % Initialize lower stroke limit. ``  `on` or `off` `` Default = ``off``.
+            'lowerLimitBound', -1, ...            % (`float`) Define lower stroke limit in m or deg. Only active if `lowerLimitSpecify` is `on` ``   `` Default = ``-1``.
+            'lowerLimitStiffness', 1e6, ...       % (`float`) Define lower limit spring stiffness, N/m or N-m/deg.  `` Default = ``1e6``.
+            'lowerLimitDamping', 1e3, ...         % (`float`) Define lower limit damping, N/m/s or N-m/deg/s.  `` Default = ``1e3``.
+            'lowerLimitTransitionRegionWidth', 1e-4) % (`float`) Define lower limit transition region, over which spring and damping values ramp up to full values. Increase for stability. m or deg. ``Default = 1e-4``
+        geometryFile            ='geometry/sphere_out.stl'; % geometry file
+        bodyMass                =1;                                             % (`float`) mass in kg, default 1
+        viz               = struct(...                       %
+            'color', [1 0 0.5], ...                            %
+            'opacity', 1)                                    % Structure defining visualization properties in either SimScape or Paraview. ``color`` (`3x1 float vector`) is defined as the body visualization color, Default = [``1 1 0``]. ``opacity`` (`integer`) is defined as the body opacity, Default = ``1``.
+        bodyparaview      = 1;                               % (`integer`) Flag for visualisation in Paraview either 0 (no) or 1 (yes). Default = ``1`` since only called in paraview.
+        bodyInertia             = [ 1 1 1];                                     % (`1 x 3 float vector`) body inertia kg-m^2, default [1 1 1]
```

Comment:
--> ``momOfInertia`` see mass comment

Reply:
Ok. I prefer keeping it somewhat clear, whether here or in the documentation, that this is the inertia/mass of the low-order lumped capacitance nodes rather than the cable itself, since the MOI of a flexible cable is complicated. That was my justification for not calling these cable.mass and cable.inertia, but this may not have been a better solution.

Reply:
Okay I see your reasoning. I guess if I was a user implementing a cable in WEC-Sim for the first time, I wouldn't know about the dummy bodies being used behind the scenes. I would probably approach this with a total mass or mass/length of the cable, but might be confused if the input file requires the mass or inertia of half the cable.